Dockerfile.okd

```diff
@@ -0,0 +1,65 @@
+# Please ensure this file is kept in sync with Dockerfile.ocp
+FROM fedora:33 AS builder
+
+WORKDIR /tmp
+
+RUN if [ $(uname -m) = "x86_64" ]; then \
+      dnf install -y gcc git make genisoimage xz-devel grub2 grub2-efi-x64 shim dosfstools mtools && \
+      dd bs=1024 count=3800 if=/dev/zero of=esp.img && \
+      mkfs.msdos -F 12 -n 'ESP_IMAGE' ./esp.img && \
+      mmd -i esp.img EFI && \
+      mmd -i esp.img EFI/BOOT && \
+      mcopy -i esp.img -v /boot/efi/EFI/BOOT/BOOTX64.EFI ::EFI/BOOT && \
+      mcopy -i esp.img -v /boot/efi/EFI/fedora/grubx64.efi ::EFI/BOOT && \
+      mdir -i esp.img ::EFI/BOOT; \
+    else \
+      touch /tmp/esp.img; \
+    fi
+
+FROM centos:8
+ENV IS_OKD=true
+
+ARG PKGS_LIST=main-packages-list.txt
+
+COPY ${PKGS_LIST} /tmp/main-packages-list.txt
+
+RUN dnf upgrade -y && \
+    dnf -y install centos-release-openstack-victoria && \
```

Comment:
is `victoria` the right version to pull from?

Reply:
In the upstream metal3 images we use the current-tripleo RDO pin, which is basically trunk plus some CI testing:

https://github.com/metal3-io/ironic-image/blob/master/prepare-image.sh#L6

The latest stable branch may also work, but I know the downstream images are also sometimes consuming newer content - @elfosardo what are your thoughts?

Reply:
Downstream images may move away from Victoria eventually (but it hasn't happened yet).

Reply:
In the list of repositories from CentOS (per https://www.rdoproject.org/what/repos/ and https://releases.openstack.org/)
- is that something like the master branch in OpenStack land? So should I pull Wallaby RPMs here instead?

Reply:
oh well, I should've scrolled down a bit further - there's an `rdo-trunk-newton-tested` repo

Reply:
ok that doesn't exist either in CentOS 8 - I'll just install the repos with the python script

Reply:
I"ve updated the PR - the only thing missing now should be the IPA build for OKD/Fedora CoreOS

Reply:
@hardys when we move away from stable/victoria  we're going to follow master directly, and in that case RDO stable will be outdated. IMHO OKD should follow current-tripleo to keep pace as you were suggesting.
For the IPA build, I guess we need to find an alternative as the ironic-images package doesn't exist upstream. Maybe also using the master images from https://tarballs.opendev.org/openstack/ironic-python-agent ? @dtantsur wdyt?